Library/Formula/ola.rb

```diff
@@ -20,14 +21,15 @@ class Ola < Formula
 
   option :universal
   option "with-ftdi", "Install FTDI USB plugin for OLA."
+  option "with-rdm-tests", "Install RDM Tests for OLA."
 
   depends_on "pkg-config" => :build
   depends_on "cppunit"
   depends_on "protobuf-c"
   depends_on "libmicrohttpd"
-  depends_on "libusb"
-  depends_on "liblo"
   depends_on "ossp-uuid"
+  depends_on "libusb" => :optional
```

Comment:
We prefer not to inflict the extra dependencies on people unconditionally unless it's something a lot of people are going to want or need.